src/main/java/ip/Duke.java

```diff
@@ -0,0 +1,339 @@
+package ip;
+
+import ip.task.Task;
+import ip.task.TaskManager;
+
+import java.util.Scanner;
+
+public class Duke {
+    /** Number of dashes used in printed horizontal line **/
+    private static final int NUM_DASHES = 60;
+    /** Prefix strings that determine the command **/
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_DONE = "done";
+    private static final String COMMAND_BYE = "bye";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+    /** Parameters for commands **/
+    private static final String PARAM_BY = "/by";
+    private static final String PARAM_AT = "/at";
+
+    public static void main(String[] args) {
+        // Print logo and greeting
+        printLogo();
+        printGreeting();
+
+        // Create TaskManager
+        TaskManager taskManager = new TaskManager();
+
+        // Scanner class for user input
+        String line;
+        Scanner in = new Scanner(System.in);
+
+        // Parsing of user input
+        do {
+            line = in.nextLine().strip();
+            execute(taskManager, line);
+        } while (!line.equals(COMMAND_BYE));
+
+        // Exit program
+        printFarewell();
+    }
+
+    private static void printHorizontalLine() {
+        System.out.println("-".repeat(NUM_DASHES));
+    }
+
+    private static void printLogo() {
+        System.out.println(
+                "    ___  ________  ________  ________\n" +
+                "   |\\  \\|\\   __  \\|\\_____  \\|\\_____  \\\n" +
+                "   \\ \\  \\ \\  \\|\\  \\\\|___/  /|\\|___/  /|\n" +
+                " __ \\ \\  \\ \\   __  \\   /  / /    /  / /\n" +
+                "|\\  \\\\_\\  \\ \\  \\ \\  \\ /  /_/__  /  /_/__\n" +
+                "\\ \\________\\ \\__\\ \\__\\\\________\\\\________\\\n" +
+                " \\|________|\\|__|\\|__|\\|_______|\\|_______|");
+    }
+
+    private static void printGreeting() {
+        printHorizontalLine();
+        System.out.println(" Hey there! The name's Jazz.\n" +
+                " What can I do for you?");
+        printHorizontalLine();
+    }
```

Comment:
Hi. Your code looks fine throughout.Just would like to suggest that
you do not have to split  lines(61-62) into two seperate lines.They dont seem to exceed the 120 char
limit given to us. Great job nonetheless!

Reply:
Yep, it would still fit. I suppose it just looks clearer for me to be on the next line, and provides some consistency with my other code that uses `\n` i.e. the logo.